pkg/k8sutil/create.go

```diff
@@ -19,12 +19,17 @@ package k8sutil
 import (
```

Comment:
The commit message content should be function documentation :wink: Or ideally, each section of commit message should be separate function with appropriate name :)